app/views/spotlight/catalog/_edit_default.html.erb

```diff
@@ -5,7 +5,7 @@
       <% current_exhibit.custom_fields.each do |field| %>
         <div class="form-group">
           <%= d.label field.field, field.label %>
-          <%= d.text_field_without_bootstrap field.field, value: document[field.field], class: 'form-control' %>
+          <%= d.text_field_without_bootstrap field.field, value: @document[field.field], class: 'form-control' %>
```

Comment:
I'm not sure why we're using `@document` instead of just `document`. If it works, I think we should prefer the local variable.